tsconfig.json

```diff
@@ -20,7 +20,7 @@
         "outDir": "dist/",
         "sourceMap": false,
         "strictNullChecks": true,
-        "target": "es5"
+        "target": "es2015"
```

Comment:
I doubt this will work... AFAIK some es2015 syntax features emitted by `tsc` are still unsupported in the versions of Node where documentalist runs. I'll double check that this is still the case... if it is, I think another transpilation step is required (and we might ask them to revert the break).

for reference, this is the reasoning they provided for the change:

> I changed the tsconfig target from es5 to es2015. This change was made in order to make the [ReflectionFlags](https://github.com/Gerrit0/typedoc/blob/strict-null-checks/src/lib/models/reflections/abstract.ts#L102) class act as a class. Since the lib property already includes es2015 libraries, and we only test on node 6+, I believe this is a justifiable change.

Reply:
Ok, let me know how/if I should test that or what I can do

Reply:
actually, it looks like all non-deprecated versions of Node 6.5+ support the ES features used here. neat! we should probably make a mention in the `"engines"` block of package.json, but otherwise g2g

Reply:
the exceptions I was thinking of all had to do with the ES module syntax, which is not widely supported in Node yet

Reply:
actually this ended up causing a blueprint bug. filed https://github.com/palantir/documentalist/issues/84, i'm looking into fixing it